cmd/minikube/cmd/root.go

```diff
@@ -117,6 +120,9 @@ func init() {
 	RootCmd.PersistentFlags().Bool(showLibmachineLogs, false, "Show logs from libmachine.")
 	RootCmd.PersistentFlags().String(username, "", "User name for the virtual machine.")
```

Comment:
Wanted to comment before on this, but this might be the better scope. "User name for the virtual machine." seems wrong and could be misinterpreted as some sort of credentials to log onto the VM. Really this should be something like "User name for virtual machine registration"